extensions/2.0/Khronos/KHR_node_visibility/schema/node.KHR_node_visibility.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_node_visibility glTF Node Extension",
+    "type": "object",
+    "description": "glTF extension that defines node's visibility.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "visible": {
+            "type": "boolean",
+            "description": "Specifies whether the node is visible.",
+            "default": true,
+            "gltf_detailedDescription": "Specifies whether the node is visible. A value of false means that the node and all its children are hidden."
```

Comment:
Nitpick

```suggestion
            "gltf_detailedDescription": "Specifies whether the node is visible. A value of false means that the node and all its descendants are hidden."
```

Reply:
I also considered to bring that up, but... it transitively applies to all relevant nodes, so this does not change the _meaning_ on a technical level. (One could argue which of them is easier to understand, conceptually, but I don't have a strong opinion there...)